src/RawRabbit.Enrichers.Polly/Middleware/BasicPublishMiddleware.cs

```diff
@@ -31,7 +31,7 @@ public BasicPublishMiddleware(IExclusiveLock exclusive, BasicPublishOptions opti
 					[RetryKey.PublishMandatory] = mandatory,
 					[RetryKey.BasicProperties] = basicProps,
 					[RetryKey.PublishBody] = body,
-				});
+				}).GetAwaiter().GetResult();			
```

Comment:
Since we're planning to go all in, I think we should make the call to this method async. That way, we get rid of the `.GetAwaiter().GetResult()`. Makes sense?